composer.json

```
@@ -30,8 +30,8 @@
         "evenement/evenement": "^3.0 || ^2.0 || ^1.0",
         "react/dns": "^1.8",
         "react/event-loop": "^1.2",
-        "react/promise": "^2.6.0 || ^1.2.1",
-        "react/promise-timer": "^1.8",
+        "react/promise": "3.x-dev as 2.999.9999",
+        "react/promise-timer": "1.x-dev as 1.999.999",
```

Could be simplified like this now that we've prepared all others components:

```suggestion
        "react/promise": "^3@dev || ^2.6 || ^1.2.1",
        "react/promise-timer": "^1.9",
```

Arguably, we may want to omit the `@dev` suffix once this is no longer WIP. This can still be tested after running `composer config minimum-stability dev`.

---

> Arguably, we may want to omit the `@dev` suffix once this is no longer WIP. This can still be tested after running `composer config minimum-stability dev`.

Testing it with `@dev` but final push will be without.